// @kierico-ui/tokens/src/colors.ts
/** colors.ts */
export const colors = {
  white: '#FFFFFF',
  black: '#000000',

  gray_100: '#E1E1E6',
  gray_200: '#C4C4CC',
  gray_300: '#8D8D99',
  gray_400: '#7C7C8A',
  gray_500: '#505059',
  gray_600: '#323238',
  gray_700: '#29292E',
  gray_800: '#202024',
  gray_900: '#121214',
  gray_950: '#09090A',

  green_light: '#00B37E',
  green_mid: '#00875F',
  green_dark: '#015F43',
  green_low: '#00291D',

  purple_light: '#996DFF',
  purple_mid: '#8257E5',
  purple_dark: '#633BBC',
  purple_low: '#271A45',

  blue_light: '#6A80FF',
  blue_mid: '#4863F7',
  blue_dark: '#3249CB',
  blue_low: '#182049',

  red_light: '#FC4737',
  red_mid: '#D73628',
  red_dark: '#AD1E12',
  red_low: '#42110D',

  success_light: '#04D361',
  success_base: '#1B873F',
  success_low: '#051B0D',

  danger_light: '#F75A68',
  danger_base: '#CC2937',
  danger_low: '#2D090C',

  warning_light: '#FBA94C',
  warning_base: '#EB8A1D',
  warning_low: '#2E1B06',

  new_light: '#1EF7D0',
  new_base: '#07847E',
  new_low: '#163840',

  cor_teste: '#c3c3c4',
}

/** https://andromeda.rocketseat.dev/?path=/docs/docs-colors--page */
